<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lunaid_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="vscore_spin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="who_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>WHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="vscore_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Score</string>
   </property>
  </widget>
  <widget class="QLabel" name="lunaid_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>LunaID</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="note_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="note_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
  </widget>
  <widget class="QLabel" name="note_label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>All Tasks</string>
   </property>
  </widget>
  <widget class="QLabel" name="collected_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Collected Tasks</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tasks_list">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="all_tasks_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>481</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="search_all_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>search all</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
